README.md

```diff
@@ -1 +1,59 @@
-# Rubrics Backend
+# Trubric-Backend
+
+www.trubric.io
+
+## Table of Contents
+
+# Trubric.io
+
+Backend API Server that supports the Single Page application that allows students to achieve measurable feedback by giving them tools for diagnostic, self-assessment through rubrics created by educators. This is project for Make School. You can reach us at through our contact below.
+
+### Authors
+- [Tassos Laumbrou](https://www.linkedin.com/in/lambrou/) - Front/Back End
+- [Alirie Gray](https://www.linkedin.com/in/alirie-gray/) - Front End/ Back End
+- [Jeff Chiu](https://www.linkedin.com/in/jeffchiu1) - Full Stack
+- [Shane Austrie] - Frontend
+
+For the Back-end side, we use Express, Node JS, Sequelize to handle our API server.
+
+
+### Development
+```
+npm install
+npm start
+```
+This will start an express app at http://localhost:8000.
+
+
+### Production Build
+```
+npm run build
+npm run build:start
+```
+This will create a production build and will start a server at http://localhost:8000/
+
+
+### Deployment
+Install the Heroku toolbelt.
+```
+heroku create myapp
+git push heroku master
+```
+
+### TODO
+- Set up a server to fix the refresh page issue (see [this](https://stackoverflow.com/questions/27928372/react-router-urls-dont-work-when-refreshing-or-writting-manually) Stack Overflow question for more information)
+- Write responsive css for mobile usage
+- Deal w/ security of API keys (and get new API keys...)
+
```

Comment:
What the heck are all these?  Perhaps add the major features that we are trying to add from trello.

Reply:
It's removed.